lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -124,16 +124,22 @@ def _supplement_text(inferred: list[Element], text: list[Element], threshold: fl
             if matched:
                 matches = []
                 full_text = []
+                font_sizes = []
                 for m in matched:
                     matches.append(m)
                     if m.text_representation:
                         full_text.append(m.text_representation)
-
+                        if m.properties.get("font_size") is not None:
```

Comment:
Replace with `if (font_size := m.properties.get("font_size"))` to reduce hash lookups